app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php

```diff
@@ -628,8 +628,12 @@ public function resizeFile($source, $keepRatio = true)
         }
         $image = $this->_imageFactory->create();
         $image->open($source);
+
         $image->keepAspectRatio($keepRatio);
-        $image->resize($this->_resizeParameters['width'], $this->_resizeParameters['height']);
+
+        list($imageWidth, $imageHeight) = $this->getResizedParams($source);
+        
+        $image->resize($imageWidth, $imageHeight);
```

Comment:
Can you maybe drop some info here, do we still need a resize even if image size stays the same?

Reply:
yes, because we create e new image in thumbnail path